### PR TITLE
fix(website): Web Deployer bugfixes

### PR DESCRIPTION
**Pull request by FuzzB0t**
This PR fixes and adds a few items on the web deployer:

- Add ability to specify custom registry for publishing
- Reenables publishing on the web deployer
- updates @safe-global/api-kit version to latest (fixes bug on SafeApiKit when querying it for executed txns)

fixes https://linear.app/usecannon/issue/CAN-186

### CODE REVIEW

File: packages/website/src/features/Deploy/PublishUtility.tsx [hunk 1]
```diff
@@ -66,9 +69,14 @@ export default function PublishUtility(props: {
 
       const [walletAddress] = await wc.data.getAddresses();
 
+
```
Comment (dbeal-eth) [inline packages/website/src/features/Deploy/PublishUtility.tsx:72]: extra line added

File: packages/website/src/hooks/safe.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import SafeApiKit from '@safe-global/api-kit';
 import { useQuery } from '@tanstack/react-query';
-import { Address, getAddress, isAddress, keccak256, stringToBytes } from 'viem';
+import { Address, getAddress, isAddress, keccak256, stringToBytes, createWalletClient, custom } from 'viem';
```
Comment (dbeal-eth) [inline packages/website/src/hooks/safe.ts:3]: lint failures are reported here

**Comment by FuzzB0t** [comment]
Getting this weird error running builder tests on builder.test.ts and provision.test.ts

image

**Comment by FuzzB0t** [comment]
> > Getting this weird error running builder tests on builder.test.ts and provision.test.ts
> > image
> 
> I also saw this but I think it's not related to this PR.

Hmmmmm on which branch did you see this in?

**Pull request merged**